sensing/autoware_pointcloud_preprocessor/schema/crop_box_filter_node.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Crop Box Filter Node",
+  "type": "object",
+  "definitions": {
+    "crop_box_filter": {
+      "type": "object",
+      "properties": {
+        "min_x": {
+          "type": "number",
+          "description": "minimum x-coordinate value for crop range in meters",
+          "default": "-1.0"
+        },
+        "min_y": {
+          "type": "number",
+          "description": "minimum y-coordinate value for crop range in meters",
+          "default": "-1.0"
+        },
+        "min_z": {
+          "type": "number",
+          "description": "minimum z-coordinate value for crop range in meters",
+          "default": "-1.0"
+        },
+        "max_x": {
+          "type": "number",
+          "description": "maximum x-coordinate value for crop range in meters",
+          "default": "1.0"
+        },
+        "max_y": {
+          "type": "number",
+          "description": "maximum y-coordinate value for crop range in meters",
+          "default": "1.0"
+        },
+        "max_z": {
+          "type": "number",
+          "description": "maximum z-coordinate value for crop range in meters",
+          "default": "1.0"
+        },
+        "negative": {
+          "type": "boolean",
+          "description": "if the value is true, filter points inside the box; otherwise, filter points outside the box",
```

Comment:
Using the work `filter` is ambiguous, even more due to how this parameter is called. Use filter in or filter out (I have already made this comment before)

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8466/commits/7f917b88277714dffb5aee82a718900bfc5caa50